src/components/Card/Card.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import {useNavigate} from 'react-router-dom';
+import {ListItemColumn, Teams, UserData} from 'types/types';
+import {CardContainer} from './Card.styled';
+
+interface CardProps {
+    id: string;
+    url?: string;
+    columns: ListItemColumn[];
+    hasNavigation?: boolean;
+    navigationProps?: UserData | Teams;
+}
+
+const Card = (props: CardProps): JSX.Element => {
+    const {
+        id,
+        columns,
+        url =  '',
+        hasNavigation = true,
+        navigationProps = null,
+    } = props;
+
+    const navigate = useNavigate();
+
+    const handleOnClick = (event: React.MouseEvent<HTMLDivElement>): void => {
```

Comment:
created a handleOnClick function, to be called in the onClick method of the CardContainer element, and specified the type of the event and method, to avoid typescript issues